knox/topologies/example-hive-nonpii.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0"?>
```

Comment:
@yunus How this file is used?

Reply:
It is an example configuration where a guest account is allowed to access wedhdfs. I have used this in tests.